delegation-toolkit/tutorials/create-custom-caveat-enforcer.md

```diff
@@ -1,31 +1,38 @@
 ---
-description: Learn how to create, deploy, and apply a custom caveat enforcer
-sidebar_position: 3
+description: Follow this tutorial to create a custom caveat enforcer for a delegation.
```

Comment:
Just a question why we are changing the sentence formation? Also, we should target deploy, and apply keywords as well.